sound/soc/sof/amd/renoir.c

```diff
@@ -44,6 +44,14 @@ const struct snd_sof_dsp_ops sof_renoir_ops = {
 
 	/* DSP core boot */
 	.run			= acp_sof_dsp_run,
+
+	/*IPC */
+	.send_msg		= acp_sof_ipc_send_msg,
+	.ipc_msg_data		= acp_sof_ipc_msg_data,
+	.ipc_pcm_params		= acp_sof_ipc_pcm_params,
+	.get_mailbox_offset	= acp_sof_ipc_get_mailbox_offset,
```

Comment:
remind me why you need to get the mailbox offset if you are not using the mailbox at all?

Reply:
Actually get_mailbox_offset is use by loader module during sof_fw_ready and hence implemented to avoid error. . Just like to clarify that we are indeed using a kind  mailbox mechanism only for Host to DSP communication but initially we have not implemented mailbox_write()/read() callbacks. Also. Moreover  acp_sof_ipc_get_mailbox_offset() actually return correct mailbox offset hardcoded in driver . We will certailny improvise mailbox related code more in near future to get it more align with SOF core.